lib/src/changes.dart

```diff
@@ -201,6 +206,9 @@ final Map<String, String> _textReplacements = {
 
   // dart:io/string_transformer.dart
   'SYSTEM_ENCODING': 'systemEncoding',
+
+  // @proxy annotation
+  '@{proxy}': '',
```

Comment:
Is this assuming that you run dartfmt afterwards? Otherwise you may want to have a match for @proxy on a line by itself, where you remove the entire line, and one where you just remove `@proxy\s*`.

Reply:
This could be extended to add some reasonable fixups for whitespace. I'd be happy to take PRs that do that. Currently however, I think the tool's in a good place wrt the amount of time we've invested in it vs. what it can do, and if putting significantly more effort into it, I'd want to do that in the analysis server via more code aware quick fixes.